riff-cli/pkg/docker/docker_utils_test.go

```diff
@@ -69,6 +74,15 @@ var _ = Describe("DockerUtils", func() {
 
 				oldPath, pathWasSet = os.LookupEnv("PATH")
 				Expect(os.Unsetenv("PATH")).To(Succeed())
+
+				oldDockerHost, dockerWasSet = os.LookupEnv("DOCKER_HOST")
+				oldDockerCert, _ = os.LookupEnv("DOCKER_CERT_PATH")
```

Comment:
This should record `dockerCertWasSet` so that the `AfterEach` can correctly reset or unset the value. As this stands, if `DOCKER_CERT_PATH` is set but `DOCKER_HOST` is not set, the test will corrupt the value of `DOCKER_CERT_PATH` (by unsetting it rather than resetting it to its original value).

I guess it's unlikely that `DOCKER_CERT_PATH` would be set without `DOCKER_HOST`, but for the sake a few lines of code, this assumption can be removed and the code made watertight.